x-pack/elastic-agent/pkg/agent/program/methods.go

```diff
@@ -112,6 +114,52 @@ func hasNamespace(env *env, args []interface{}) (interface{}, error) {
 	return true, nil
 }
 
+// hasAny the methods take a list of possible keys where at least one of those keys must exist.
+func hasAny(env *env, args []interface{}) (interface{}, error) {
```

Comment:
Do we have somewhere tests for this?

Reply:
We have in-direct tests just like the other methods in this file, being that it is used in the endpoint.yml spec and if it didn't work those tests would fail.